docs/moonray/howto/install.md

```diff
@@ -0,0 +1,3 @@
+```{include} ../../src/snap/howto/install/snap.md
+
```

Comment:
<img width="1197" alt="image" src="https://github.com/user-attachments/assets/ed2d1ece-de0a-4788-892f-2302c9bf0bb6">
The note's Installing leads us to the how-to page, there's probably a page missing could you please add a dummy missing page.

Reply:
Same goes for the other links on this page. Do we need to bring in the channel page and bootstrap ref guide?

Reply:
I think we will either need to put conditional blocks in there or harmonise the directory structure first. I have just added a todo note for now - bringing in those pages will bring in more unresolvable links right now